### PR TITLE
Fe 8 calendar detail insert

### PR DESCRIPTION
**Pull request by Kyandroo**

### CODE REVIEW

File: frontend/src/components/calendar/CalendarCard/CalendarCard.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        src={img}
+        alt={`thumbnail picture of "${name}" calendar`}
+      />
+
+      <div className="calendar-card__content">
+        <span className="heading heading--sm calendar-card__title">{name}</span>
+
+        <ul className={`calendar-card__user-list`}>
+          <li>
+            <Icon src={userImgSrc} alt={userImgAlt} />
```
Comment (Jefdeniel) [inline frontend/src/components/calendar/CalendarCard/CalendarCard.tsx:26]: add fallback

Comment (Jefdeniel) [reply frontend/src/components/calendar/CalendarCard/CalendarCard.tsx:26]: allready loop over a couple of items to make it easier in the future

File: frontend/src/components/layout/Navigation/CalendarNavigation.tsx [hunk 2]
```diff
@@ -68,50 +54,73 @@ const CalendarNavigation = ({
...
+          </Col>
+        </Row>
+
+        <Row className={`mb-large`}>
+          <SearchBar />
         </Row>
 
+        <Row>
+          <span className={`heading heading--sm clr-primary-300 mb-base`}>
+            {t('general:navigation.title'.toString())}
```
Comment (Jefdeniel) [inline frontend/src/components/layout/Navigation/CalendarNavigation.tsx:88]: is toString() necessary? Doesn't it also work without?

**Review by Jefdeniel — CHANGES_REQUESTED** [review]

File: frontend/src/components/ui/Icon/SvgIcons.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+interface Props {
+  className?: string;
+}
+
+const NotificationIcon = ({ className }: Props) => {
+  return (
+    <div className={`${className}`}>
+      <svg
+        width="13"
```
[the remark this answers is not in the document]
Comment (Kyandroo) [reply frontend/src/components/ui/Icon/SvgIcons.tsx:11]: Possible, but what when later on we want to use them somewhere else?

File: frontend/src/components/ui/EventCard/Card.scss
[the remark this answers is not in the document]
Comment (Kyandroo) [reply frontend/src/components/ui/EventCard/Card.scss:1]: For the moment, still has to be styled

File: frontend/src/pages/Calendar/Calendar.tsx [hunk 4]
```diff
@@ -30,7 +33,30 @@ const Calendar = () => {
     return <Spinner />;
   }
 
-  return <div>{!isLoading ? 'Calendar' : 'Loading...'}</div>;
+  return (
+    <>
+      <div>{!isLoading ? '' : 'Loading...'}</div>
```
[the remark this answers is not in the document]
Comment (Kyandroo) [reply frontend/src/pages/Calendar/Calendar.tsx:38]: Do you mean like this?
<div>{!isLoading ? '' : <Spinner />}</div>

[the remark this answers is not in the document]
Comment (Kyandroo) [reply frontend/src/pages/Calendar/Calendar.tsx:38]: Do you mean like this?
`<div>{!isLoading` ? '' : <Spinner />}</div>`

[the remark this answers is not in the document]
Comment (Kyandroo) [reply frontend/src/pages/Calendar/Calendar.tsx:38]: Do you mean changing the 'loading...' to the Spinner component?

**Pull request merged**